ciphey/basemods/Crackers/__init__.py

```diff
@@ -1 +1 @@
-from . import caesar, vigenere, xor_single, xorcrypt, XandY
```

Comment:
Side note: I think GitHub will  only auto-close the issue if you use the word "fixes" maybe?

So instead of "Solves #346" it's "Fixes #346"

Reply:
Ahh, yes. I will **fix** that : ) 
Looked at what keywords that are usable, and it is 
`[close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved]`